docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrGit_Source_Code.xml

```diff
@@ -0,0 +1,610 @@
+      https://github.com/gituser/gch-ecldemo-d#version1. You could also
+      include the -v option to provide more verbose output including the
+      details of the Git requests.</para>
+    </sect3>
+
+    <sect3 id="ECLandECLCCGitOptions">
+      <title>ECL and ECLCC Git Options</title>
+
+      <para>These command line options have been added to the ECL and ECLCC
+      commands.</para>
+
+      <para>--<emphasis role="bold">defaultgitprefix</emphasis> This command
+      line option changes the default prefix that is added to relative
+      packages references. The default can also be configured using the
+      environment variable ECLCC_DEFAULT_GITPREFIX. Otherwise It defaults to
+      "https://github.com/".</para>
+
+      <para>--<emphasis role="bold">fetchrepos</emphasis> Setting this option
+      tells whether external repositories that have not been cloned locally
+      should be fetched. This defaults to true in 8.6.x. It may be useful to
+      set this option to false if all external repositories are mapped to
+      local directories to verify if they are being redirected
+      correctly.</para>
+
+      <para>--<emphasis role="bold">updaterepos</emphasis> Updates external
+      repositories that have previously been fetched locally. This option
+      defaults to true. It is useful to set this option to false if you are
+      working in a situation with no access to the external repositories, or
+      to avoid the overhead of checking for changes if you know there aren't
+      any.</para>
+
+      <para><emphasis role="bold">ECLCC_ECLREPO_PATH</emphasis> The directory
+      the external repositories are cloned to. On a client machine this
+      defaults to: &lt;home&gt;/.HPCCSystems/repos (or
+      %APPDATA%\HPCCSystems\repos on windows). You can delete the contents of
+      this directory to force a clean download of all repositories.</para>
+    </sect3>
+
+    <sect3 id="GitHelmChartConfigurationOptions">
+      <title>Helm Chart Configuration Options</title>
+
+      <para>These are Helm chart options for configuring Git values for cloud
+      deployments. The following values are now supported for configuring the
+      use of Git within Helm charts for HPCC Systems cloud deployments.</para>
+
+      <para><emphasis role="bold">eclccserver.gitUsername</emphasis> -
+      Provides the Git user name</para>
+
+      <para><emphasis role="bold">secrets.git </emphasis>- Define the
+      secrets.git to allow repositories to be shared between queries, to be
+      able to cache and share the cloned packages between instances.</para>
+
+      <para><emphasis role="bold">eclccserver.gitPlane</emphasis> - This
+      options allows the user to define the storage plane that external
+      packages are cloned to. If the option is not supplied, the default is
+      the first storage plane with a category of Git - otherwise ECLCC Server
+      uses the first storage plane with a category of dll.</para>
```

Comment:
Worth including example sections from a helm chart to make it clear.  Possibly a comment see examples in the authentication section below.

Reply:
please review the example I provided, I don't think I wrote it correctly